app/views/mw_interactives/edit.html.haml

```diff
@@ -15,6 +15,12 @@
     .clear
     This width and height will be used to maintain a fixed aspect ratio for the interactive.
   %br
+  = field_set_tag 'Pre-flight URL' do
+    = f.text_area :preflight_url, size: "45x4"
+  %div{:style => "margin-bottom:5px;"}
+    %em Warning: A Pre-flight URL automatically enables Click to Play.
+    Please provide an image_url to use a Pre-flight URL.
```

Comment:
Does this image have to live on the same host as the interactive?  If so, I think we need more instructions for the author.